Plugins/Flow.Launcher.Plugin.Explorer/Languages/ko.xaml

```diff
@@ -1,112 +1,116 @@
-﻿<ResourceDictionary
-    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
-    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-    xmlns:system="clr-namespace:System;assembly=mscorlib">
+﻿<?xml version="1.0"?>
+<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:system="clr-namespace:System;assembly=mscorlib">
 
     <!--  Dialogues  -->
-    <system:String x:Key="plugin_explorer_make_selection_warning">항목을 먼저 선택하세요</system:String>
+    <system:String x:Key="plugin_explorer_make_selection_warning">Please make a selection first</system:String>
     <system:String x:Key="plugin_explorer_select_folder_link_warning">폴더 링크를 선택하세요</system:String>
-    <system:String x:Key="plugin_explorer_delete_folder_link">{0} - 삭제하시겠습니까?</system:String>
-    <system:String x:Key="plugin_explorer_deletefolderconfirm">이 폴더를 영구적으로 삭제하시겠습니까?</system:String>
-    <system:String x:Key="plugin_explorer_deletefileconfirm">이 파일을 영구적으로 삭제하시겠습니까?</system:String>
-    <system:String x:Key="plugin_explorer_deletefilefoldersuccess">삭제 완료</system:String>
-    <system:String x:Key="plugin_explorer_deletefilefoldersuccess_detail">{0} - 성공적으로 삭제했습니다.</system:String>
-    <system:String x:Key="plugin_explorer_globalActionKeywordInvalid">글로벌 액션 키워드는 너무 많은 결과를 불러오게 될 수 있습니다. 특정한 액션 키워드를 선택하세요.</system:String>
-    <system:String x:Key="plugin_explorer_quickaccess_globalActionKeywordInvalid">빠른 접근은 글로벌 액션키워드로 사용할 수 없습니다. 특정한 액션 키워드를 선택하세요.</system:String>
-    <system:String x:Key="plugin_explorer_windowsSearchServiceNotRunning">윈도우색인 검색에 필요한 서비스가 실행되어 있지 않습니다.</system:String>
-    <system:String x:Key="plugin_explorer_windowsSearchServiceFix">이 문제를 해결하려면 Windows Search Service를 시작하세요. 이 경고를 제거하려면 여기를 선택하세요.</system:String>
-    <system:String x:Key="plugin_explorer_alternative">경고 메시지가 꺼졌습니다. 파일 및 폴더 검색을 위한 대안으로 Everything 플러그인을 설치하시겠습니까?{0}{0}Everything 플러그인을 설치하려면 '예'를 선택하고, 반환하려면 '아니오'를 선택하십시오</system:String>
+    <system:String x:Key="plugin_explorer_delete_folder_link">Are you sure you want to delete {0}?</system:String>
+    <system:String x:Key="plugin_explorer_deletefolderconfirm">Are you sure you want to permanently delete this folder?</system:String>
+    <system:String x:Key="plugin_explorer_deletefileconfirm">Are you sure you want to permanently delete this file?</system:String>

```

Comment:
@onesounds check translations please

Reply:
It's a lot to fix now. Ignore it in this patch. (I translated it for the test PR.) I will work on Crowdin later.